tests/FilterTest.php

```diff
@@ -243,18 +325,25 @@ public function testLessThanOrEqualMismatches()
         $this->assertFalse(Filter::match($lessThanOrEqual, $this->row(2)));
     }
 
-    public function testEqualWithWildcardMatches()
+    public function testEqualWithWildcardMismatches()
     {
         $equal = Filter::equal('service', '*icinga*');
 
-        $this->assertTrue(Filter::match($equal, $this->row(1)));
+        $this->assertFalse(Filter::match($equal, $this->row(1)));
     }
 
-    public function testEqualWithWildcardMismatches()
+    public function testSimilarWithWildcardMatches()
     {
-        $equal = Filter::equal('service', '*nagios*');
+        $similar = Filter::similar('service', '*icinga*');
 
-        $this->assertFalse(Filter::match($equal, $this->row(1)));
+        $this->assertTrue(Filter::match($similar, $this->row(1)));
+    }
+
+    public function testSimilarWithWildcardMismatches()
+    {
+        $similar = Filter::similar('service', '*nagios*');
+
+        $this->assertFalse(Filter::match($similar, $this->row(1)));
     }
 
     public function testUnequalWithWildcardMatches()
```

Comment:
This needs an update now. `*nagios*` didn't match previously not because the wildcards were ignored, but because it just didn't match. To test that the wildcards are ignored now, this needs to check for `*icinga*`.